<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_motifs</class>
 <widget class="QDialog" name="dlg_motifs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>596</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="UpTableWidget" name="MotifsupTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>320</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Detailsframe">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>21</y>
     <width>230</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>160</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="UpLineEdit" name="MotifupLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>45</number>
    </property>
   </widget>
   <widget class="UpLabel" name="MotifupLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motif</string>
    </property>
   </widget>
   <widget class="QLabel" name="RaccourciupLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Raccourci</string>
    </property>
   </widget>
   <widget class="UpLineEdit" name="RaccourciupLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="UpCheckBox" name="DefautupCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Par d√©faut</string>
    </property>
   </widget>
   <widget class="UpCheckBox" name="UtiliserupCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Utiliser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CouleurpushButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>92</y>
      <width>220</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Modifier la couleur de fond</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpTableWidget</class>
   <extends>QTableWidget</extends>
   <header>uptablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>UpLineEdit</class>
   <extends>QLineEdit</extends>
   <header>uplineedit.h</header>
  </customwidget>
  <customwidget>
   <class>UpLabel</class>
   <extends>QLabel</extends>
   <header>uplabel.h</header>
  </customwidget>
  <customwidget>
   <class>UpCheckBox</class>
   <extends>QCheckBox</extends>
   <header>upcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
